### PR TITLE
Added endpoint-slice.md

### PR DESCRIPTION
**Pull request by CloudShiva7cr**
### Description
Added the endpoint-slice.md a translation from english to Japanese. Pls review and let me know if any changes needed to make. Thank you!!!

### Issue
Translation of English (endpoint-slice.md )--> Japanese (endpoint-slice.md)

Closes: #

### CODE REVIEW

File: container-env-variables.md
Comment (Okabe-Junya) [inline container-env-variables.md:1]: please delete this file

File: content/ja/docs/reference/glossary/endpoint-slice.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: エンドポイントスライス (EndpointSlice)
+id: endpoint-slice
+date: 2024-10-10
+full_link: /docs/concepts/services-networking/endpoint-slices/
+short_description: >
+  Kubernetesリソースでネットワークエンドポイントをグループ化する方法
+
+
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/endpoint-slice.md:9]:
```suggestion
```

File: content/ja/docs/reference/glossary/endpoint-slice.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: エンドポイントスライス (EndpointSlice)
+id: endpoint-slice
+date: 2024-10-10
+full_link: /docs/concepts/services-networking/endpoint-slices/
+short_description: >
+  Kubernetesリソースでネットワークエンドポイントをグループ化する方法
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/endpoint-slice.md:7]: If you want to move this PR forward, please translate _**all**_ the sentences properly into Japanese.

File: content/ja/docs/reference/glossary/endpoint-slice.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+tags:
+- networking
+---
+ Kubernetesリソースでネットワークエンドポイントをグループ化する方法
+
+
+
+<!--more-->
+
+ネットワークエンドポイントをグループ化するための、スケーラブルかつ拡張可能な方法です。これらは、各ノードでkube-proxyによってネットワークルートを確立するために使用されます。
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/endpoint-slice.md:21]: ditto

**Review by Okabe-Junya — COMMENTED** [review]
Some comments left. please have a look, thanks

File: content/ja/docs/reference/glossary/endpoint-slice.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+title: エンドポイントスライス (EndpointSlice)
+id: endpoint-slice
+date: 2024-10-10
```
Comment (Okabe-Junya) [inline content/ja/docs/reference/glossary/endpoint-slice.md:4]: you don't need update this date (plz copy from original)

File: container-env-variables.md
Comment (CloudShiva7cr) [reply container-env-variables.md:1]: deleted pls review

**Review by CloudShiva7cr — COMMENTED** [review]
Pls review and let me know!

File: content/ja/docs/reference/glossary/endpoint-slice.md (hunk 2 above)
Comment (CloudShiva7cr) [reply content/ja/docs/reference/glossary/endpoint-slice.md]: sure, have changed as you said pls review it once ! thanks.

**Comment by CloudShiva7cr** [comment]
cc @Okabe-Junya @t-inu Pls review once and let me know chnages!

**Comment by CloudShiva7cr** [comment]
I will make necessary changes and raise PR!!

**Pull request closed without merging**